### ISSUE TITLE
treeTable: Header row is one pixel smaller in width than the other rows when sticky headers is enabled

### ISSUE THREAD

**Issue by ken-han** [primefaces/primefaces#4390]
Test repo --> https://github.com/ken-han/primefaces-header-bug

## 1) Environment
- PrimeFaces version: 6.2
NOTE: Same result in primefaces version 6.2.13

- Application server + version: Wildfly 13
- Affected browsers: Chrome

## 2) Expected behavior

A neat looking table with the header row and content rows alligned
![image](https://user-images.githubusercontent.com/46558687/50965138-6c58f380-14d1-11e9-9ff4-a3c2efe57d5e.png)

## 3) Actual behavior

A not so neat looking table with the header row being one pixel smaller in width than the content rows. If it isn't immediately noticeable, zoom in a little bit. Once you see it, you can't unsee it.

![image](https://user-images.githubusercontent.com/46558687/50965184-8a265880-14d1-11e9-92f9-3bb81e4f84b1.png)

## 4) Steps to reproduce

Creating a treetable with **stickyHeader=true**. 
See test repo --> https://github.com/ken-han/primefaces-header-bug

**Comment by melloware** [comment]
@ken-han Any debugging you have done as to why or provide a PR to help us fix it?

**Comment by ken-han** [comment]
@melloware We haven't been able to find out why this happens. We "fixed" this by simply adding a single pixel to the table header with javascript.

Ofcourse since this bug only happens with chrome, in firefox the table header is a pixel too large.

**Comment by melloware** [comment]
I just tested this again with PF8.0 in the latest Chrome and the issue appears to have resolved itself.  I also noticed as I zoom in our out with Chrome I can see it shift slighly depending on the zoom factor but I think that is just an artifact of zooming.  

[pf-4390.zip](https://github.com/primefaces/primefaces/files/4430085/pf-4390.zip)

I am going to close this issue for now. We can re-open if you don't see my example attached shows its fixed.
